File: client/Crypto.java
import javax.crypto.SecretKey;

public class Crypto {
	
	public static void encrypt(String message, String rsaPublicKey) {
		AES aes = new AES();
		
	}
	
	public static void decrypt(String jsonObj, String rsaPrivateKey) {
		
		SecretKey key = null;
		AES aes = new AES(key);
	}

}

File: server/Crypto.php
<?php 
//var $message = 'Hi';
//var $rsaPublicKey = '/home/ubuntu/public.pem';

class Crypto {

	public static function encrypt($message, $rsaPublicKey) {
		$rsa = new RSA($rsaPublicKey);
		$rsa->loadPublicKey();
		//var $aes = new AES();
		//echo $message;
	}

	public static function decrypt($jsonObj, $rsaPrivateKey) {

	}

}

?>